neo4j-cypher-dsl/src/main/java/org/neo4j/cypherdsl/core/Functions.java

```diff
@@ -79,7 +71,7 @@ public static FunctionInvocation id(@NotNull Node node) {
 	@NotNull @Contract(pure = true)
 	@Deprecated(since = "2023.3.0")
 	@SuppressWarnings({ "squid:S1133" }) // Yes, I promise, this will be removed at some point, but not yet.
-	public static FunctionInvocation id(@NotNull Relationship relationship) {
+	static FunctionInvocation id(@NotNull Relationship relationship) {
```

Comment:
This could get already removed because there is no access to this method anymore, in contrast to the `id(Node node)` version that is used by the `AbstractNode`.